.circleci/config.yml

```diff
@@ -1772,18 +1795,27 @@ workflows:
           docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
           requires: ['op-stack-go-docker-build']
           save_image_tag: <<pipeline.git.revision>> # for devnet later
+      - docker-build:
+          name: da-server-docker-build
+          docker_name: da-server
+          docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
+          requires: ['op-stack-go-docker-build']
+          save_image_tag: <<pipeline.git.revision>> # for devnet later
       - cannon-prestate:
           requires:
             - go-mod-download
       - devnet:
+          name: devnet<< matrix.fpac >>
           matrix:
             parameters:
-              fpac: ["legacy", "fault-proofs"]
+              # TODO(10142): We need to re-enable "-fault-proofs" in the future
```

Comment:
Fixed in https://github.com/ethereum-optimism/optimism/pull/10144

Reply:
PR approved. Will rebase once it's merged.